### DISCUSSION TITLE
Microsoft Edge console log errors

### DISCUSSION THREAD

**Discussion by dotnetshadow** [Dotnet-Boxed/Templates#37] [Q&A]
Hi there,

Awesome project, but I did notice some console log errors with regards to IIS Express when using the ASP.NET MVC 6 Boilerplate (v2.4.0) 

HTTP404: NOT FOUND - The server has not found anything matching the requested URI (Uniform Resource Identifier).
(XHR): GET - http://localhost:34498/bower_components/bootstrap-sass/assets/stylesheets/_bootstrap.scss

HTTP404: NOT FOUND - The server has not found anything matching the requested URI (Uniform Resource Identifier).
(XHR): GET - http://localhost:34498/bower_components/bootstrap-sass/assets/stylesheets/bootstrap/_normalize.scss

![capture](https://cloud.githubusercontent.com/assets/296247/10654544/83b2de4c-78b7-11e5-8b1e-2079538bd6df.PNG)

This doesn't happen in Firefox or Chrome, although if I navigate to one of the urls in question it doesn't show anything just blank, whereas Microsoft Edge browser throws a 404 error

**Comment by RehanSaeed** [comment]
Thanks, was it you that emailed me this error? Could be a problem with the source map or something. I'll investigate when I get some time. Maybe even a bug in Edge.

**Comment by dotnetshadow** [comment]
Yes I mentioned it on the visual studio gallery website.

**Comment by RehanSaeed** [comment]
I just tried this with 2.6.0 and it seems fine. Can you confirm? Not sure if its because I updated a  few NPM packages or if there was an Edge bug fix in windows update.

**Comment by dotnetshadow** [comment]
I still have the errors, but I just upgraded using the nuget packages. Do I have to make any other code changes? 

![capture3](https://cloud.githubusercontent.com/assets/296247/11023343/54cabb4c-86ca-11e5-9500-1695d213d7bc.PNG)
![capture4](https://cloud.githubusercontent.com/assets/296247/11023344/54cc8da0-86ca-11e5-82aa-db9ccbc290a9.PNG)

**Comment by dotnetshadow** [comment]
Actually I created a new web application with 2.6.0 and it seems the errors have gone away. But if I upgrade the project then the error is still there. 

Not sure what are the changes? I think I assumed that when upgrading the nuget packages it would upgrade various controllers etc but it looks like it only upgrades the nuget packages and doesn't touch any other source files. So it's hard to keep up with upgrading from version to version if the source files aren't being updated. i.e. like ISitemapService wasn't changed in 2.7 but couldn't see the change when I did it through the nuget packages.

**Comment by RehanSaeed** [comment]
Check the NPM packages. An update may have solved this issue.
